test/apps/cloudAppClient/manager-eventRequest.test.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+var test = require('tape')
+var helper = require('../../helper')
+var EventEmitter = require('events')
+var Manager = require(`${helper.paths.apps}/cloudappclient/manager.js`)
+var Skill = require(`${helper.paths.apps}/cloudappclient/skill`)
+var eventRequestMap = require(`${helper.paths.apps}/cloudappclient/eventRequestMap.json`)
+
+class EventRequest extends EventEmitter {
+  ttsEvent (name, appId, itemId, cb) {
+    var res = '{}'
+    cb(res)
+    this.emit(`tts-${appId}-${name}`, itemId)
+  }
+
+  mediaEvent (name, appId, extra, cb) {
+    var res = '{}'
+    cb(res)
+    this.emit(`media-${appId}-${name}`, extra)
+  }
+}
+
+test('test manager-eventRequest: media cancel', (t) => {
+  t.plan(1)
+  var skill = new Skill({}, {}, {
+    appId: 'testAppId',
+    response: {
+      action: {}
+    }
+  })
+  var eventBus = new EventRequest()
+  var pm = new Manager(null, Skill)
+  // In order to achieve the purpose of our testing, we need to intercept the real request.
+  pm.eventRequest = eventBus
+  pm.skills = [skill]
+
+  var name = eventRequestMap['media']['cancel']
```

Comment:
use `_.get` to simplify?

Reply:
Just want a simple value, don't worry about the variable being null